src/main/java/se/nbis/lega/inbox/sftp/InboxSftpEventListener.java

```diff
@@ -258,6 +259,10 @@ protected String getFilePath(Path path, String username) {
         }
         log.debug("POSIX filepath is {} for user {}", key, username);
 
+//        if (inboxFSPath != null && inboxFSPath != "") {
+//            return inboxFSPath + "/" + key    ;
+//        }
+
```

Comment:
Commented out lines should be removed.

Reply:
actually that should have been there my mistake